dictionary.txt

```diff
@@ -913,12 +937,9 @@ _rp	₎
 \frac56	⅚	
 \frac18	⅛	
 \frac38	⅜	
-\frac58	⅝	
+\frac58 ⅝ 
 \frac78	⅞	
 \co	℅	
--	−	
```

Comment:
Put this line back, it's a shortcut for converting hyphen (`-`) to longer minus sign (`−`)